test/test_util/react_util.dart

```diff
@@ -156,8 +156,8 @@ Element renderAndGetDom(dynamic component, {bool addToRenderedQueue: true}) {
   return findDomNode(render(component, addToRenderedQueue: addToRenderedQueue));
 }
 
-/// Renders a React component or builder into a detached node and returns the associtated Dart component.
-react.Component renderAndGetComponent(dynamic component) => getDartComponent(render(component));
+/// Renders a React component or builder into a detached node and returns the associated Dart component.
+UiComponent renderAndGetComponent(dynamic component) => getDartComponent(render(component)) as UiComponent;
```

Comment:
Why is this changed to `UiComponent`? It's possible to use this to render a non-over_react component, so we should leave it as `react.Component`.